### DISCUSSION TITLE
Welcome to svelte-fa Discussions!

### DISCUSSION THREAD

**Discussion by Cweili** [Cweili/svelte-fa#70] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jmroon** [comment]
Stumbled upon this package when I tried to add FA to my SvelteKit project today.

Absolutely beautiful work. Tree shaking and customizable? All I could ever want. Very simple to use, and very elegant.

I may contribute in the future if there's anything I feel is lacking, but so far, no need!

**Reply by Cweili** [reply]
Thanks for your encouragement! Enjoy it!
